Automattic/woocommerce-payments
# Add admin notice to warn about deactivating WC Subscriptions

#### Changes proposed in this Pull Request

This PR contains the changes to introduce a confirmation admin notice when users click the deactivate WC Subscriptions plugin.

Project: pdjTHR-35-p2
Notice feedback discussion: pdjTHR-aF-p2 

<img width="1013" alt="Screen Shot 2021-10-07 at 3 53 14 pm" src="https://user-images.githubusercontent.com/8490476/136327477-704feb09-4325-4504-b52a-8c7c98a0590b.png">

#### Testing instructions

1. Install and activate the WC Subscriptions plugin.
2. Make sure the WCPay Subscriptions feature is switched on via **WCPay Dev tools**. https://d.pr/HjtrhT
3. Go to the **Plugins** admin screen. 
4. Click the link to **Deactivate** WooCommerce Subscriptions. 
5. On this branch you should be prompted with a popup notice.
6. Clicking **Yes, deactivate WooCommerce Subscriptions** will deactivate the plugin, cancel will close the modal and subscriptions will remain active.

If you disable the WCPay Subscriptions feature via **WCPay Dev tools**, deactivating the plugin doesn't lead to this prompt. The result of this means that if your site isn't eligible for WC Pay Subscriptions (feature flag isn't on) then this prompt isn't shown - the plugin just deactivates as normal. 

**Important note:** WCPay Subscriptions will be enabled by default on eligible stores. Later we'll introduce a setting to disable it.

The learn more links to a doc that hasn't been written but the URL was suggested by Brent in pdjTHR-aF-p2 

-------------------


**Post merge**




closes #2963
